packages/ui-kit/src/components/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import type { Meta, StoryObj } from '@storybook/react'
```

Comment:
Should this be a Story and a component we expose to our customers? AFAICT we are not planning to expose the autocomplete itself but rather its usage (filters, time range picker, etc) and then in each integration we enable customizing the component's styles / customizing every Select in the theme provider

I don't see it bad exposing it directly though so lmk if you think it's worthy

Reply:
@felipecadavid  @markandrus @hasancolak @acossta 
I just use the story for the component development. There are pros (customers will be able to build their Dashboard by using just our UI Kit) and cons (more dev efforts, maintenance costs) in exposing generic components (buttons, inputs, autocomplete, etc). It's worth the discussion.

Reply:
Hmm. I think for now, we should hide the auto-complete for now. Let's get experience using it first (i.e., let's build the components we plan to actually expose), before adding it to our public API.